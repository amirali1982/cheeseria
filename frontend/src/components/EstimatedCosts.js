import {Component} from "react";

class EstimatedCosts extends Component {
    constructor(props) {
        super(props);
    }

    render() {
        return (
            <div><h4>Estimated Cost: {this.props.value}</h4></div>);

    }
}

export default EstimatedCosts;