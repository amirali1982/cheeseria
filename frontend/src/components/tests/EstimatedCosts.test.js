import renderer from 'react-test-renderer';
import EstimatedCosts from "../EstimatedCosts";

it('Testing Estimated Costs', () => {
    const component = renderer.create(
        <EstimatedCosts value={10}/>
    )

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    expect(tree.children["0"].children[1]).toBe("10");
});