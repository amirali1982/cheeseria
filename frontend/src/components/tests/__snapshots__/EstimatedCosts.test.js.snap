// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Estimated Costs 1`] = `
<div>
  <h4>
    Estimated Cost: 
    10
  </h4>
</div>
`;
